src/hooks/identity-hooks.ts

```diff
@@ -0,0 +1,18 @@
+import { useQuery, type UseQueryOptions } from "@tanstack/react-query";
```

Comment:
you could probably toss this into `/src/features/identity/hooks.ts` 

i would expect a `src/hooks` folder to contain reusable helpers across all domains (for example like the `useFetch` you were planning on doing)